src/richie/apps/core/context_processors.py

```diff
@@ -86,3 +92,93 @@ def site_metas(request):
 
     context["FRONTEND_CONTEXT"] = json.dumps(context["FRONTEND_CONTEXT"])
     return context
+
+
+class WebAnalyticsContextProcessor:
+    """
+    Context processor to add Web Analytics tracking information to Richie CMS templates and
+    frontend.
+    """
+
+    def context_processor(self, request: HttpRequest) -> dict:
+        """
+        Real implementation of the context processor for the Web Analytics core app sub-module
+        """
+        context = {}
+        if hasattr(request, "current_page"):
+            # load web analytics settings to the context
+            if getattr(settings, "WEB_ANALYTICS_ID", None):
+                context["WEB_ANALYTICS_ID"] = settings.WEB_ANALYTICS_ID
+            if getattr(settings, "WEB_ANALYTICS_LOCATION", None):
+                context["WEB_ANALYTICS_LOCATION"] = settings.WEB_ANALYTICS_LOCATION
```

Comment:
In fact I think the default should be set here, not in the sandbox setting file so they follow to people's project.
```suggestion
            context["WEB_ANALYTICS_LOCATION"] = getattr(settings, "WEB_ANALYTICS_LOCATION", "head")
```

Reply:
Done.